.flaskenv

```diff
@@ -2,4 +2,5 @@ FLASK_APP=app.py
 FLASK_ENV=development
 FLASK_RUN_PORT=5000
 FLASK_RUN_HOST=localhost
-DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:7890/fsd_app_store_test
+DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/application_store
+USE_LOCAL_DATA=True
```

Comment:
Can we revert these changes? seem unrelated.

Reply:
sorry! reverted back, forgot to change it after testing locally 👍🏾